### DISCUSSION TITLE
[Projects Beta] Don't show private issues if the viewer doesn't have permission to see them

### DISCUSSION THREAD

**Discussion by markapruett** [community/community#16444] [Projects and Issues]
Currently when there issues in the new proejcts beta view that a viewer doesn't have access to see it shows this:
<img width="922" alt="Screen Shot 2022-05-12 at 8 53 59 AM" src="https://user-images.githubusercontent.com/67116121/168091487-0e4b508c-59e8-475a-a14f-6c9632b21fab.png">

It would help if those issues weren't visible at all in the list, or at least give a setting to toggle visibility on and off.

Thanks!

**Comment by mariorod** [comment]
Hi Mark and thanks for the feedback. We call the above functionality "issue redaction" and the core reasoning is that customers wanted "one source of truth" for how many items are in the project. We will continue to monitor feedback to see if we need to change our approach.
